scala-package/core/src/test/java/org/apache/mxnet/javaapi/NDArrayTest.java

```diff
@@ -86,15 +86,15 @@ public void testGenerated(){
         NDArray$ NDArray = NDArray$.MODULE$;
```

Comment:
Do you still need this line?

Reply:
Yes, it didn't save the world of the method like`NDArray.<method_name>` popping out